conda/build.sh

```diff
@@ -8,6 +7,8 @@ mkdir -p 'build' && cd 'build'
 cmake \
   -DPYTHON_EXECUTABLE="$CONDA_PREFIX/bin/python" \
   -DCMAKE_INSTALL_PREFIX="$CONDA_PREFIX" \
+  -DCMAKE_OSX_DEPLOYMENT_TARGET=$OSX_VERSION \
+  -DCMAKE_OSX_SYSROOT="/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX$OSX_VERSION.sdk" \
```

Comment:
I don't like having to do this, it might be worth considering `source` (supported by travis in travis.yaml) and binding to output of `xcrun --sdk macosx --show-sdk-path` prior to the miniconda installation. 

Getting the osx major/minor version is also acheveable via something like `sw_version`